docs/yaml/builtins/meson.yaml

```diff
@@ -152,6 +152,21 @@ methods:
       - `vs2022`
       - `xcode`
 
+  - name: build_options
+    returns: str
+    since: 0.64.0
```

Comment:
The next version will be `1.0.0` rather than `0.64.0`.

As this is new functionality this MR should also have a release note snippet.

Reply:
I don't see any updated release notes in the current tree, how do I add the snippet?

Reply:
Look in docs/markdown/snippets

Reply:
This still says 0.64.0, needs to be changed.

Reply:
What's the next release number, so that I can change it correctly?

Reply:
Updated the version string to 1.1.0. Let me know if that's correct.